HomeTask1_2/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace HomeTask1_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter 9 symbols for ISBN");
+            string isbnInput = Console.ReadLine();
+
+            int weight = 10;
+            int result = 0;
+
+            for (int i = 0; i < isbnInput.Length; i++)
+            {
+                int convertValue = isbnInput[i] - '0';
+
+                result += weight * convertValue;
+
+                weight--;
+            }
+
```

Comment:
Not correct checkDigit, please check (Bug), for examlpe, for "123456789" right check digit is X, not 1